articles/nat-gateway/troubleshoot-nat-connectivity.md

```diff
@@ -331,6 +321,16 @@ To create a Private Link, see the following Quickstart guides to get started:
 
 * [Create a Private Link](../private-link/create-private-link-service-portal.md)
 
+## More troubleshooting guidance
```

Comment:
The troubleshooting guidance where it is currently located is directly after the troubleshooting steps.  Placing it after the best practices section could cause customers to bounce out of the article if they aren't concerned about Outbound best practices and miss this guidance.

Reply:
agreed, the "more troubleshooting guidance" should directly follow the troubleshooting steps. It serves as a - if you went through all these steps and your investigation is still inconclusive, here's the steps to follow.

Reply:
@vikedesai please move the "More troubleshooting guidance" section with the "Extra network captures" back to where it was originally - after the troubleshooting steps section.